server/src/entity/arrow.rs

```diff
@@ -0,0 +1,62 @@
+use crate::entity;
+use crate::entity::{ComponentSerializer, EntityId, SpawnPacketCreator, Velocity};
+use crate::game::Game;
+use crate::physics::PhysicsBuilder;
+use crate::util::{degrees_to_stops, protocol_velocity};
+use feather_core::entity::{ArrowEntityData, BaseEntityData, EntityData};
+use feather_core::network::packet::implementation::SpawnObject;
+use feather_core::{Packet, Position, Vec3d};
+use fecs::{EntityRef, World};
+use uuid::Uuid;
+
+pub fn spawn_arrow(game: &mut Game, world: &mut World, velocity: glm::DVec3, position: Position) {
```

Comment:
To be consistent with the item implementation, this should be called `create` and return an `EntityBuilder` for adding further components.